backend/controllers/controller_helm_applications.go

```diff
@@ -0,0 +1,83 @@
+package controllers
+
+import (
+	"fmt"
+	"github.com/ZPI-2024-25/KubernetesAccessManager/helm"
+	"github.com/ZPI-2024-25/KubernetesAccessManager/models"
+	"net/http"
+)
+
+func GetHelmReleaseController(w http.ResponseWriter, r *http.Request) {
+	handleHelmOperation(w, r, models.Read, func(releaseName, namespace string) (interface{}, *models.ModelError) {
+		return helm.GetHelmRelease(releaseName, namespace)
+	})
+}
+
+func GetHelmReleaseHistoryController(w http.ResponseWriter, r *http.Request) {
+	handleHelmOperation(w, r, models.Update, func(releaseName, namespace string) (interface{}, *models.ModelError) {
```

Comment:
```suggestion
	handleHelmOperation(w, r, models.Read, func(releaseName, namespace string) (interface{}, *models.ModelError) {
```
Shouldn't that be models.Update

Reply:
I'm not sure. I think Release History will be used in rollback, so it should be Update. But this needs discussion with @samuelus.

Reply:
I will change it, but it still feels iffy.